x-pack/plugins/observability_solution/observability/public/pages/alerts/alerts.tsx

```diff
@@ -72,6 +74,29 @@ function InternalAlertsPage() {
 
   const filteredRuleTypes = useGetFilteredRuleTypes();
 
+  const { setScreenContext } = observabilityAIAssistant?.service || {};
+
+  const ruleTypesWithDescriptions = useGetAvailableRulesWithDescriptions();
+
+  useEffect(() => {
+    return setScreenContext?.({
+      screenDescription: `The rule types that are available are: ${JSON.stringify(
```

Comment:
Arguably this should be done via `data`, but it's a little awkward to do (map over each rule with `description` 'An available rule is ${rule}`?). This works well and passes less tokens.

Reply:
`data` has `name`, `description` and `value`. you need the latter I think instead of `description`. The advantage of `data` is that these tokens don't get sent over by default unless it's below x tokens.

Reply:
Updated